<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forwarding_Window</class>
 <widget class="QWidget" name="Forwarding_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>转发</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>收件人：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_receiver">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>15</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>主题：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edt_theme">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>64</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>正文：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edt_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>转发邮件信息：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edt_forwarding">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_forward">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>转发</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
